sstables/sstable_directory.cc

```diff
@@ -590,12 +590,12 @@ future<std::pair<sstring, sstring>> sstable_directory::create_pending_deletion_l
             close_out.close_now();
 
             auto dir_f = open_directory(pending_delete_dir).get0();
+            auto close_dir = deferred_close(dir_f);
```

Comment:
nit: please spell check commit message.

Also, it may be more elegant to replace `dir_f.close().get()` with `close_dir.close_now()` so it becomes explicit and evident before the log message.

Reply:
Hm... OK